src/styles/components/breadcrumbs.less

```diff
@@ -142,3 +142,11 @@ Breadcrumbs
     }
   }
 }
+
+.breadcrumb-style-headline {
+  margin-bottom: @base-spacing-unit * 2/3;
+  .h4 {
+    font-size: @base-spacing-unit * .65;
```

Comment:
I wonder if we should instead declare a font-size rather than using the base-spacing-unit for this...?